apps/database-new/components/MainTagWrapper.tsx

```diff
@@ -0,0 +1,23 @@
+'use client'
+import { usePathname } from 'next/navigation'
+import { cn } from 'ui'
+
+interface MainTagWrapperProps {
+  children: React.ReactNode
+}
+
+const MainTagWrapper = ({ children }: MainTagWrapperProps) => {
+  const pathname = usePathname()
+  const isFullHeight = pathname.startsWith('/thread') || pathname === '/'
+
+  return (
+    <main
+      role="main"
+      className={cn('w-full flex flex-col grow', isFullHeight && 'h-[calc(100vh-115px)]')}
```

Comment:
> 115px

magic number. we don't need to do this do we ?

Reply:
Can just have a outer container of h-screen, and then child elements just fill the space